smithy-aws-traits/src/main/resources/META-INF/smithy/aws.experimental.smithy

```diff
@@ -0,0 +1,70 @@
+$version: "1.0"
```

Comment:
Each shape/member in this file should have documentation. Will need specification updates for the traits as well.